### PR TITLE
Retention at rank 4

### PR DESCRIPTION
**Pull request by kianenigma**

### CODE REVIEW

**Comment by kianenigma** [comment]
Will open the proposal once I get at least one approval from @bkchr, or other members.

File: evidence/kianenigma/0002-2024-12-retention-rank-4.md
Comment (anaelleltd) [inline evidence/kianenigma/0002-2024-12-retention-rank-4.md:45]: |IV| 60% | 90% | 51 votes out of 107, leading to a 48% rate, starting from ref 148 onwards, which was my retention in June |  |

Comment (anaelleltd) [reply evidence/kianenigma/0002-2024-12-retention-rank-4.md:45]: Please do not edit the thresholds. Only fill in the 'Member's voting activities" and "Comment" columns with your data.

File: evidence/kianenigma/0002-2024-12-retention-rank-4.md
Comment (joepetrowski) [inline evidence/kianenigma/0002-2024-12-retention-rank-4.md:52]:
```suggestion
- I will strive to improve my voting rate, and apply for a promotion to Rank 5 in June 2025. I will still retain my part time salary only. My main intention to promote myself would be to eligible to vote on more proposals, and remove the burdon on the current rank 5+ members for promotions to rank 3+, and so on. 
```

**Review by anaelleltd — APPROVED** [review]

**Review by ljuanto — APPROVED** [review]

File: evidence/kianenigma/0002-2024-12-retention-rank-4.md
Comment (ljuanto) [inline evidence/kianenigma/0002-2024-12-retention-rank-4.md]: 60% | 90%

**Review by ljuanto — APPROVED** [review]

**Pull request merged**
